org.gnome.Screenshot.json

```diff
@@ -0,0 +1,72 @@
+{
+  "app-id": "org.gnome.Screenshot",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.28",
+  "sdk": "org.gnome.Sdk",
+  "command": "gnome-screenshot-wrapper",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=wayland",
+    "--talk-name=org.gnome.Shell.Screenshot",
+    "--filesystem=xdg-run/dconf",
+    "--filesystem=~/.config/dconf:ro",
+    "--talk-name=ca.desrt.dconf",
+    "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
+    "--filesystem=home"
```

Comment:
`xdg-pics` is enough IMO.

Reply:
@bilelmoussaoui fwiw is it possible to use the "Portal-ized" filechooser in the current UI? Or would that need a redesign.

Reply:
Oh turns out filechooserbuttons don't support selecting files to write, weird.

Reply:
@nedrichards Honestly, I don't follow up the development of Screenshot that closely.  I have no idea if there was any work before to have a Flatpak package. I have contacted upstream to see if they are interested to maintain it & see what they think about the current limitations.